src/waveform/renderers/waveformmark.h

```diff
@@ -23,31 +31,27 @@ class WaveformMark {
     WaveformMark& operator=(const WaveformMark&) = delete;
 
     // Enable swapping
-    void swap(WaveformMark& that) {
-        std::swap(m_pPointCos, that.m_pPointCos);
-        std::swap(m_properties, that.m_properties);
-        std::swap(m_iHotCue, that.m_iHotCue);
-        std::swap(m_image, that.m_image);
-    }
-    friend void swap(WaveformMark& lhs, WaveformMark& rhs) {
-        lhs.swap(rhs);
-    }
-
-    void reset(int hotCue = kDefaultHotCue);
-
-    void setup(const QString& group, const QDomNode& node,
-               const SkinContext& context,
-               const WaveformSignalColors& signalColors);
-
-    std::unique_ptr<ControlProxy> m_pPointCos;
+    // Swapping is done in the destructor
```

Comment:
I removed the swapping function, so may be I forgot to remove this comment, so I remove it